tests/unit/test_forum_models.py

```diff
@@ -517,9 +517,9 @@ def test_post_delete(topic):
     post_last.delete()
 
     # That was a bit trickier..
```

Comment:
Indeed it was, looks like the old logic was the initial post didn't account towards anything, the logic changes here makes it do just that.